build/ansible/roles/nginx/files/conf.d/pmm.conf

```diff
@@ -133,7 +135,7 @@
     }
 
     # Grafana
-    rewrite ^/$ $scheme://$http_host/graph/;
+    rewrite ^/$ /graph/ permanent;
```

Comment:
Let's not make it permanent, it might change in future

Reply:
"permanent" stands for 301 and gets cached by browser for speed. But whenever we change the `/graph/` part, the cache will be busted immediately. So that should be fine, I beleieve.